samples/CommunityToolkit.Maui.Sample/App.xaml

```diff
@@ -66,7 +66,7 @@
             </Style>
 
             <!--  Values  -->
-            <Thickness x:Key="ContentPadding">20, 0</Thickness>
+            <maui:Thickness x:Key="ContentPadding">20, 0</maui:Thickness>
```

Comment:
Just a double check, do we need to specify this explicitly?

Reply:
That shouldn’t depends on IDE.

Reply:
Yup! I totally agree. It's definitely a bug.

We should be able to remove it once VS for Mac releases improved .NET MAUI support